README.md

```diff
@@ -14,15 +31,19 @@ While Swift-Kuery is not an Object-Relational Mapping (ORM), it provides a great
 Swift-Kuery is easy to learn, consumable framework that comes with a set of [implemented plugins](#list-of-plugins).
 
 ## Table of Contents
-* [Example](#example)
+* [Examples](#examples)
 * [SQL Injection Prevention using Parameterization](#sql-injection-prevention-using-parameterization)
 * [Prepared Statements](#prepared-statements)
 * [Schema Management](#schema-management)
 * [Query Examples](#query-examples)
 * [List of plugins](#list-of-plugins)
 * [License](#license)
 
-## Example
+## Examples
+A Swift-Kuery tutorial on adding a PostgreSQL database to a foodTracker app is available [here](https://github.com/IBM/FoodTrackerBackend/blob/master/AddDatabase.md).
```

Comment:
Line 100 - 
please reword to - (added words an and then)
"Suppose we want to retrieve the average grades for courses with an average greater than 90 and then sort the results by the average ascending. Here is the SQL for this query:"